pandas/core/indexes/datetimelike.py

```diff
@@ -430,6 +430,7 @@ def min(self, axis=None, *args, **kwargs):
         --------
         numpy.ndarray.min
         """
+        nv.validate_minmax_axis(axis)
         nv.validate_min(args, kwargs)
```

Comment:
Is there reason not to add the axis validation to the existing `validate_min` ?

Reply:
exactly I don't want *another* function, rather you can simply check this in side the function which is already there.

Reply:
Done.  I'm not wild about the fact that the nv.validate_(min|max|argmin|argmax) functions now implicitly assume they are only being called on 1-dim objects, but at least the assumption is correct for now.

Reply:
Hmm, yeah, that makes sense. 
And adding them in a single validation is actually also mixing two kinds of validation: validation of arguments that are purely for numpy compat (things like `out`), opposed to validation of valid arguments for pandas (`axis` in the Series and Index methods is also there for consistency with DataFrame, than for compat with numpy)